api.yaml

```diff
@@ -46,31 +46,21 @@ info:
     name: Apache 2.0
     url: 'https://www.apache.org/licenses/LICENSE-2.0.html'
 paths:
-  /healthcheck:
-    get:
-      description: "Liveness check."
-      responses:
-        "200":
-          description: "Success."
-      x-transportd:
-        backend: app
-  /{name}:
-    get:
-      description: An API Call.
-      parameters:
-        - name: "name"
-          in: "path"
-          description: "Name of person to greet."
-          required: true
-          schema:
-            type: "string"
+  /2015-03-31/functions/{functionName}/invocations:
```

Comment:
So this would be the path to use if you wanted your gateway to act as a proxy with the lambda SDK. In this case, the lambda component below is redundant. The lambda component is specific to serverful-gateway, and it is intended to adapt a RESTful API to a lambda function.

The goal of this file is to serve as an example configuration i.e. it won't be the file we use for our internal service. If you want to keep an example of how someone might use this as a real lambda, but also take advantage of the accesslogging/metrics provided by transportd, then you could leave this here, but I'd recommend changing the base image to transportd. Otherwise, I'd recommend preserving a simple REST API e.g. /filter that gets mapped to your lambda via the serverful-gateway component.